clink/lua/src/settings_api.cpp

```diff
@@ -21,7 +21,7 @@ extern setting_bool g_lua_strict;
 /// -ver:   1.0.0
 /// -arg:   name:string
 /// -arg:   [descriptive:boolean]
-/// -ret:   boolean | string | integer | nil
+/// -ret:   boolean | string | nil
```

Comment:
This change is inaccurate.  Integer settings return an integer value.

For example,
```lua
print(type(settings.get("clink.update_interval")))
```

That prints "number" because integers are stored as numbers in Lua (IEEE 754 double precision floating point).  Internally Lua only has a number type, not an integer type.  However, 52 bit integer values can be expressed in a double precision floating point number.

"integer" in the these refers to the fact that any non-integer number argument will be coerced into an integer by stripping the fractional part, and any returned integer is a number value that is guaranteed to have no fractional part.